go.mod

```diff
@@ -322,7 +322,7 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.7.0-sdk-v0.46.7
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.8.0-sdk-0.46.7
```

Comment:
apologies for this, there should be a v here, but even after deleting the old misspelled tag and creating one with the proper name, go modules fails to resolve this. I've tried resetting go modules locally, so I don't think its that. Not sure why doing this wouldn't fix it for me, as that has worked in the past with deleting and recreating misspelled releases.